src/subclass/application.rs

```diff
@@ -281,6 +288,17 @@ impl<T: ApplicationImpl + ObjectImpl> ApplicationImplExt for T {
             f(application.to_glib_none().0)
         }
     }
+
+    fn parent_handle_local_options(&self, application: &Application, options: &VariantDict) -> i32 {
+        unsafe {
+            let data = self.get_type_data();
+            let parent_class = data.as_ref().get_parent_class() as *mut gio_sys::GApplicationClass;
+            let f = (*parent_class)
+                .handle_local_options
+                .expect("No parent class implementation for \"startup\"");
```

Comment:
Instead of exploding, return `-1` if there's none. That's what should be the normal behavior from reading the C code.